From 87ef84d994646f2b81a0edda6a5db2681c46e0a0 Mon Sep 17 00:00:00 2001
From: Brandon Philips <brandon@ifup.org>
Date: Tue, 19 Apr 2011 10:13:45 -0700
Subject: [PATCH] test/misc.test: rm f from the top of the test

f might exist before this test is ran. remove it first so the test
always starts from the same state.

Upstream-commit: a1efce0a9b02514d8063be57b45708f17486a406
Signed-off-by: Kamil Dudka <kdudka@redhat.com>
---
 test/misc.test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/misc.test b/test/misc.test
index 0e3c174..2af902b 100644
--- a/test/misc.test
+++ b/test/misc.test
@@ -3,6 +3,7 @@ Pretty comprehensive ACL tests.
 This must be run on a filesystem with ACL support. Also, you will need
 two dummy users (bin and daemon) and a dummy group (daemon).
  
+	$ rm -f f
 	$ umask 027
 	$ touch f
 	  
-- 
2.7.4

